src/utils/rounded.jsx

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+export default function rounded(x, y, w, h, cornerRadius) {
+  console.log(cornerRadius)
+  var retval;
+  retval  = "M" + (x + cornerRadius.topLeft) + "," + y;
+  retval += "h" + (w - cornerRadius.topLeft - cornerRadius.topRight);
+  if (cornerRadius.topRight != 0) { retval += "a" + cornerRadius.topRight + "," + cornerRadius.topRight + " 0 0 1 " + cornerRadius.topRight + "," + cornerRadius.topRight; }
+  else { retval += "h" + cornerRadius.topRight; retval += "v" + cornerRadius.topRight; }
+  retval += "v" + (h - cornerRadius.topRight - cornerRadius.bottomRight);
+  if (cornerRadius.bottomRight != 0) { retval += "a" + cornerRadius.bottomRight + "," + cornerRadius.bottomRight + " 0 0 1 " + -cornerRadius.bottomRight + "," + cornerRadius.bottomRight; }
```

Comment:
In general, using a conditional off of the positive case is clearer than the negative. You should favor
`if (bottomRight === 0) {} else {}` over `if (bottomRight != 0) {} else {}`
and, of course, full equality (`===` or `!==`) is better than 'truthy' (`==` or `!=`) unless you have a particular reason.